pyanaconda/modules/payloads/payload/payload_base_interface.py

```diff
@@ -50,6 +51,19 @@ def Type(self) -> Str:
         """
         return self.implementation.type.value
 
+    @property
+    def SidePayload(self) -> ObjPath:
+        """Side payload attached to this payload.
+
+        This payload will be started after the main payload is started.
```

Comment:
Is it really "after the main payload is *started*"? Or rather handled / finished / installed ?

Reply:
It should be always "after" the main payload. At least in theory the point here is that side payload is somehow dependent on the main payload.

Reply:
I mean if it shouldn't rather be after the main payload finished (instead of started, ie running the main and side payload is serial, not concurrent).

Reply:
hmm, this could be tricky. Let's try to re-phrase.

Reply:
I've tried to improve the documentation strings for side payloads in general.